cogs/customreactions.py

```diff
@@ -0,0 +1,1253 @@
+                    pass
+
+                if number < 1 or number > len(current_list):
+                    if proposals:
+                        title = "Current custom reaction proposals\n" \
+                                "Click on {} to approve, reject, edit, or " \
+                                "see more information on one of them " \
+                                "(Previous attempt received invalid input " \
+                                "or timed out)".format(EMOJI["ok"])
+                    else:
+                        title = "Current custom reactions\nClick on {} " \
+                                "to edit or see more information on one of " \
+                                "them (Previous attempt received invalid " \
+                                "input or timed out)".format(EMOJI["ok"])
+                    p = Pages(ctx,
+                              msg=message,
+                              item_list=reaction_dict,
+                              title=title,
+                              display_option=(2, 10),
+                              editable_content_emoji=EMOJI["ok"],
+                              return_user_on_edit=True)
+                else:
+                    left = await information_on_react(message, current_list,
+                                                      number, proposals)
+                    if left:
+                        return True
+                    if proposals:
+                        title = "Current custom reaction proposals\n" \
+                                "Click on {} to approve, reject, edit, or " \
+                                "see more information on one of them"\
+                                .format(EMOJI["ok"])
+                    else:
+                        title = "Current custom reactions\nClick on {} to " \
+                                "edit or see more information on one of them"\
+                                .format(EMOJI["ok"])
+
+                    # update dictionary since a custom reaction might have been
+                    # modified
+                    if proposals:
+                        current_list = self.proposal_list
+                    else:
+                        current_list = self.reaction_list
+
+                    if not current_list:
+                        if proposals:
+                            title = 'There are currently no custom ' \
+                                    'reaction proposals in this server'
+                        else:
+                            title = 'There are currently no custom ' \
+                                    'reactions in this server'
+                        embed = discord.Embed(title=title)
+                        await message.edit(embed=embed, delete_after=60)
+                        return
+
+                    index_list = [
+                        '[{}]'.format(i + 1) for i in range(len(current_list))
```

Comment:
same index thing; if this is a commonly used concept maybe can make a fn for generating these lists of indexes based on a given iterable or whatever